### PR TITLE
Update image sizes on the published web page

### PR DESCRIPTION
**Pull request by ihhub**
related to #5623

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -7,10 +7,11 @@ improvements in the gameplay, graphics and logic (including support for high-res
 and UI improvements), breathing new life into one of the most addictive turn-based strategies.
 
 <p align="center">
-    <img src="images/screenshots/screenshot_world_map.png?raw=true" width="410"> <img src="images/screenshots/screenshot_castle.png?raw=true" width="410">
+    <img src="docs/images/screenshots/screenshot_world_map.png?raw=true" width="820">
```
Comment (oleg-derevenetz) [inline docs/README.md:10]: Since this `README.md` is in `docs`, we don't need to add `docs/` here and on the next similar line.

**Review by oleg-derevenetz — CHANGES_REQUESTED** [review]
Hi @ihhub it seems that file paths are now wrong.

File: docs/README.md (hunk 1 above)
Comment (ihhub) [reply docs/README.md]: Yes, you were right. I fixed it.

**Review by zenseii — COMMENTED** [review]
Hi, @ihhub.
I feel that the picture showing the adventure map is too small, at least when viewed from my PC:

<details>

image

</details>

Also the third and last picture is larger than the other two above. Do we scale these images at integers so that the pixels are as sharp as possible?

**Comment by ihhub** [comment]
Hi @zenseii , this pull request addresses these issues. The third screenshot is bigger because of fixed size of the image set in html format.

**Review by zenseii — APPROVED** [review]

**Pull request merged**